cla_public/static-src/javascripts/modules/currencyInputValidation.js

```diff
@@ -0,0 +1,42 @@
+// ($(this).val() && Number($(this).val()) > 0)
+
```

Comment:
Unless you think this is still needed for some reason, delete it.  If you want it left, can you say why (in the comment, or another comment next to it)